jquery.signaturepad.js

```diff
@@ -528,14 +528,14 @@ function SignaturePad (selector, options) {
     }
 
     if (typeItNumChars > oldLength && typed.outerWidth() > element.width) {
-      while (typed.outerWidth() > element.width) {
+      while (typeItCurrentFontSize > 4 && typed.outerWidth() > element.width) {
```

Comment:
Would this work if we make it `> 0` instead? I'm not sure what the best option is, but it seems like we should allow it so long as it's a positive number.

Reply:
I was thinking that it should be at least visible. I can change this to zero if that would be best.